ui-asset-management/api/svc-asset-related-assets/extension/clients/elasticsearch_client.go

```
@@ -104,11 +104,49 @@ func (c *ElasticSearchClient) FetchRelatedAssets(ctx context.Context, docTypes [
 
+func (c *ElasticSearchClient) FetchParentRelatedAssets(ctx context.Context, tenantId, ag, assetId string) (*map[string]interface{}, error) {
+
+	query := buildParentRelatedAssetsQuery(assetId)
+	esRequest := map[string]interface{}{
+		"size":    1000,
+		"query":   query,
+		"_source": []string{docId},
+	}
+
+	var buffer bytes.Buffer
+	json.NewEncoder(&buffer).Encode(esRequest)
```

golangci-lint (1.62.2)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)